treeherder/model/sql/jobs.json

```diff
@@ -47,6 +47,11 @@
 
             "sql":"DELETE FROM result_set WHERE id IN (REP0)",
             "host": "master_host"
+        },
+        "cycle_performance_artifact":{
+
+            "sql":"DELETE FROM performance_artifact WHERE job_id IN (REP0)",
+            "host": "master_host"
```

Comment:
s/host/host_type/ when you rebase (we've been using the wrong value everywhere, which is now fixed on master: bug 1125464).